book.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "skatejs",
+  "plugins": [ "jsfiddle"],
+  "pluginsConfig": {
+    "jsfiddle":{
+      "type":"script",
+      "tabs":["result","js", "html","css"],
+      "height": "250",
+      "width": "500",
+      "fontColor": "00FF00"
```

Comment:
After experimenting I found that it looked best matching the regular paragraph color - hope that's okay. Otherwise we can easily change it.